src/components/10-atoms/carousel/child.scss

```diff
@@ -15,3 +15,25 @@ axa-carousel {
     @include defaultAttributesChildElements();
   }
 }
+
+axa-carousel[visible="0"] > *:first-child,
+axa-carousel[visible="1"] > *:first-child + *,
+axa-carousel[visible="2"] > *:first-child + * + *,
+axa-carousel[visible="3"] > *:first-child + * + * + *,
+axa-carousel[visible="4"] > *:first-child + * + * + * + *,
+axa-carousel[visible="5"] > *:first-child + * + * + * + * + *,
+axa-carousel[visible="6"] > *:first-child + * + * + * + * + * + *,
+axa-carousel[visible="7"] > *:first-child + * + * + * + * + * + * + *,
+axa-carousel[visible="8"] > *:first-child + * + * + * + * + * + * + * + *,
+axa-carousel[visible="9"] > *:first-child + * + * + * + * + * + * + * + * + * {
```

Comment:
Oh Tannenbaum, oh Tannenbaum,.... :-D

Reply:
Thats ok for a quick solution. But to hard-limit it to 10 because of these css rules feels wrong. Is there no solution to be dynamic? On the other hand.... it is the carousel. The ugliest Webelement. So why not ;)

Reply:
I agree with danyball. I'd rather prefer a dynamic solution to generate this CSS, than limit it to 10 just because we don't want an ugly stylesheet.

Reply:
There is no unlimited dynamic solution because CSS does not have variables in selectors.

The change to controlling visibility via a single numeric attribute is well-motivated, because the mode=all functionality could not be realized efficiently with the old clumsy way of controlling slide visibility.